File: marti/Model/VintageModel/ILtaskImage.m
//
//  ILtaskImage.m
//  marti
//
//  Created by Paul Legault Local on 11-12-09.
//  Copyright (c) 2011 __MyCompanyName__. All rights reserved.
//

#import "ILtaskImage.h"
#import "ILtask.h"


@implementation ILtaskImage

@dynamic taskImage_data;
@dynamic taskImage_flags;
@dynamic taskImage_modifiedDate;
@dynamic taskImage_version;
@dynamic belongToTask;
@dynamic taskpictogramid;

-(void) setPictogramid:(NSString*)pid{
    self.taskpictogramid = pid;
}

@end

File: marti/Model/VintageModelImporter/ILtaskImage.swift
//
//  ILtaskImage.swift
//  
//
//  Created by Marco Guilmette on 2015-08-20.
//
//

import Foundation
import CoreData

class ILtaskImage: NSManagedObject {

    @NSManaged var taskImage_data: NSData
    @NSManaged var taskImage_flags: String
    @NSManaged var taskImage_modifiedDate: NSDate
    @NSManaged var taskImage_version: NSNumber
    @NSManaged var belongToTask: ILtask

}
